### DISCUSSION TITLE
Structure of React Native Document

### DISCUSSION THREAD

**Discussion by codetricity** [camera360developers/theta-client-guide#5] [Ideas]
# title  (use single hash)

picture

![image](https://user-images.githubusercontent.com/5799860/223506620-8bef9bb4-128e-4dea-b480-f1e8c5a2bbfd.png)

## overview

what the document covers

* generic react native tools and software install on Windows, 
* theta-client build and make available to demo-react-native
* build of demo-react-native
* test with THETA X on emulator and physical device (include screenshots)

note that document only covers react native windows install.  will include Mac workflow in the future

## resources

* link to official demo
* links to relevant videos for react native

## test on THETA X

<img width="101" alt="image" src="https://user-images.githubusercontent.com/5799860/223508566-e15b5059-0768-4ae9-9842-9857c72b7da6.png">

## summary and next steps

**Comment by AirTechWick** [comment]
Looks good, will check in if need to change anything
